applications/ray/kuberay-tpu-webhook/Makefile

```diff
@@ -24,6 +24,10 @@ uninstall:
 # Deploy the webhook in-cluster
 deploy:
 	kubectl apply -f deployments/
+
+# Deploy the webhook with verbose logging
+deploy-verbose:
+	kubectl apply -f deployments/ --v 1
```

Comment:
I think the --v=1 here applies to the verbosity of kubectl and not the webhook. The `--v` change needs to happen in the deployment manifest when you pass arguments to the webhook binary

Reply:
I think since we're primarily using `kubectl logs` to view the logs, passing `--v` here still works. When I test it out I can view that `deploy` and `deploy-verbose` result in the different logs printing. I can change it so that users just need to edit the deployment manifest for the more verbose logs if that's preferable/better practice though.

Reply:
Done, I added in the `--v` flag to the Deployment, users can enable verbose logging by setting it to 1.